src/extrinsic/staking.rs

```diff
@@ -16,235 +16,224 @@
 */
 
 //! Extrinsics for `pallet-staking`.
+//! https://polkadot.js.org/docs/substrate/extrinsics#staking
 
-use super::common::*;
+use super::{AssignExtrinsicTypes, ExtrinsicFor};
 use crate::{rpc::Request, Api};
 use ac_compose_macros::compose_extrinsic;
-use ac_primitives::{
-	BalancesConfig, CallIndex, ExtrinsicParams, RewardDestination, SignExtrinsic, StakingConfig,
-	UncheckedExtrinsicV4,
-};
-use codec::{Compact, Encode};
-use serde::de::DeserializeOwned;
-use sp_runtime::traits::GetRuntimeBlockType;
-
-const STAKING_MODULE: &str = "Staking";
-const STAKING_BOND: &str = "bond";
-const STAKING_BOND_EXTRA: &str = "bond_extra";
-const STAKING_UNBOND: &str = "unbond";
-const STAKING_REBOND: &str = "rebond";
-const STAKING_WITHDRAW_UNBONDED: &str = "withdraw_unbonded";
-const STAKING_NOMINATE: &str = "nominate";
-const STAKING_CHILL: &str = "chill";
-const STAKING_SET_CONTROLLER: &str = "set_controller";
+use ac_primitives::{CallIndex, ExtrinsicParams, RewardDestination, SignExtrinsic, StakingConfig};
+use codec::{Compact, Decode, Encode};
+
+const MODULE: &str = "Staking";
```

Comment:
I am not sure if I agree with the plain `Module` name. I would prefer to name `STAKING_MODULE`, what was your reasoning?

Reply:
Redundancy. The file is already called `staking.rs`. Why would everything need a Staking prefix?

Reply:
Alright, I see now that it is private, so I think it is ok. However, module is just so overly generic so I am still a bit against it.

Reply:
Kay kay, I'll give in this time. As thanks for the solution above 😉

Reply:
Readded the name. But for the MODULE only. For the calls, I still think that's an overkill.